tools/scope_upload_classification.py

```diff
@@ -11,7 +11,6 @@
 from tools import scope_manage_annotation
 from datetime import datetime
 
-MAX_ATTEMPTS = 10
```

Comment:
Should this be imported as a default from the config or something?

Reply:
I think that makes more sense - I'll find a spot in the config to add these defaults.

Reply:
@mcoughlin Just pushed those changes. I'd also like to undo #78 by reading the protocol and host from the config file again. The multiple times where I've used Scope code to interact with my localhost instance of SkyPortal have highlighted the convenience of being able to switch between protocols/hosts just by editing the config file.